.github/workflows/Documentation.yml

```diff
@@ -15,14 +15,17 @@ jobs:
       - uses: julia-actions/setup-julia@latest
         with:
           version: '1'
+      - name: Install xvfb and OpenGL libraries
+        run: sudo apt-get update && sudo apt-get install -y xorg-dev mesa-utils xvfb libgl1 freeglut3-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev libxext-dev
       - name: Install dependencies
         run: julia --project=docs/ -e 'ENV["JULIA_PKG_SERVER"] = ""; using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
       - name: Build and deploy
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # For authentication with GitHub Actions token
           DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }} # For authentication with SSH deploy key
           GKSwstype: "100" # https://discourse.julialang.org/t/generation-of-documentation-fails-qt-qpa-xcb-could-not-connect-to-display/60988
-        run: julia --project=docs/ --code-coverage=user docs/make.jl
+        run: |
+          DISPLAY=:0 xvfb-run -s '-screen 0 1024x768x24' julia --project=docs/ --code-coverage=user docs/make.jl
```

Comment:
What practical impact does this have? Does it mean all our docs are now being built under the assumption they live in a virtual 1024x768 framebuffer?

Reply:
No, the use of `xvfb` with `-screen 0 1024x768x24` does not mean all the plots rendered during doc build will be constrained to that resolution, if that's what you mean. This setup only applies during the CI build to provide a virtual display for `GLMakie` plots, which require `OpenGL`, but also doesn't dictate the final resolution of the `GLMakie` plots, which instead are determined by the plot settings themselves.

Plots rendered by `Plots.jl` (using GR) or `CairoMakie.jl` aren't affected, cause these libraries use software-based rendering that doesn't depend on an `OpenGL` display. Their output is generated independently of the `xvfb` virtual framebuffer, so that those final doc artifacts are unaffected by this temporary display configuration.

Reply:
However, if you'd rather not implement this change in the the docs CI, another option is I can just use `CairoMakie` as the backend for all the inline rendered plots, but make the presented code blocks look like they use `GLMakie`. 

But this would obviously break some of the spirit of making the plots dynamically to ensure `GLMakie` functionality in the tutorial.

Reply:
No that's fine. I just wanted to understand the code.

If tests pass is this ready to merge on your end?